.changeset/early-peaches-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/canon': patch
```

Comment:
Big breaking change and needs to be marked as such

Reply:
I changed it to a minor and added the Breaking prefix following the other PR. Since the code was actually not accessible by anyone, nobody will have to migrate to the new component.